### PR TITLE
Upgrade expect-test to 1.1

### PR DESCRIPTION
**Pull request by Jesse-Bakker**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -386,11 +386,11 @@ dependencies = [
...
 [[package]]
 name = "expect-test"
-version = "1.0.2"
+version = "1.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "eab78d4c727fa879de40951ffc01ed781a231b28786b19d6bfbdc0d1ec92eb3e"
+checksum = "2300477aab3a378f2ca00a4fbd4dc713654ab7ed790e4017493cb33656280633"
 dependencies = [
- "difference",
+ "dissimilar",
```
Comment (matklad) [inline Cargo.lock:393]: I wonder what keeps difference in tree?

Comment (Jesse-Bakker) [reply Cargo.lock:393]: proc_macro_serve and test_utils have it in their Cargo.toml. Seems like only test_utils is actually using it

Comment (Jesse-Bakker) [reply Cargo.lock:393]: If you want to I can send a patch to either:
 - use expect-test in test_utils
 - copy the formatting impl from expect-test to test_utils and use dissimilar there

Comment (matklad) [reply Cargo.lock:393]: I think migrating to expect_test is better (but I didn't look at the code)

**Pull request merged**